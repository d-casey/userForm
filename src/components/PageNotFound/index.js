import React from 'react'
import { Link } from 'react-router-dom'

const PageNotFound = () => {
  return (
    <div>
      <p data-testid='not-found-text'>That page was not found. Why not head back to the <a href='/'>Home</a></p>
    </div>
  )
}

export default PageNotFound
