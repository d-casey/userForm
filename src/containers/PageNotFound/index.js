import React from 'react'
import { Link } from 'react-router-dom'

const PageNotFound = () => {
  return (
    <div>
      <p>That page was not found. Why not head back to the <a href="/">Home</a></p>
    </div>
  )
}

export default PageNotFound
