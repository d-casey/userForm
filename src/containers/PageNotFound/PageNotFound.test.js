import React from 'react'
import { create } from 'react-test-renderer'
import PageNotFound from '.'

describe('<PageNotFound /> container', () => {
  const component = create(
    <PageNotFound />
  )
  it('renders the correct content', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })
})
