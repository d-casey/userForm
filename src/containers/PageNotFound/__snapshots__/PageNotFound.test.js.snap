// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> container renders the correct content 1`] = `
<div>
  <p>
    That page was not found. Why not head back to the 
    <a
      href="/"
    >
      Home
    </a>
  </p>
</div>
`;
